src/Linters/NoEmptyStatementsLinter.php

```diff
@@ -0,0 +1,52 @@
+<?hh // strict
+/*
+ *  Copyright (c) 2017-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the MIT license found in the
+ *  LICENSE file in the root directory of this source tree.
+ *
+ */
+
+namespace Facebook\HHAST\Linters;
+
+use type Facebook\HHAST\{
+  ExpressionStatement,
+  EditableNode,
+  EditableList,
+};
+
+final class NoEmptyStatementsLinter
+  extends AutoFixingASTLinter<ExpressionStatement> {
+  <<__Override>>
+  protected static function getTargetType(): classname<ExpressionStatement> {
+    return ExpressionStatement::class;
+  }
+
+  <<__Override>>
+  public function getLintErrorForNode(
+    ExpressionStatement $stmt,
+    vec<EditableNode> $_parents,
+  ): ?ASTLintError<ExpressionStatement> {
+
+    $expr = $stmt->getExpression();
+    if ($expr === null) {
+          return new ASTLintError(
+      $this,
+      'Delete this empty statement',
```

Comment:
Sorry if I've not followed this completely, but I aim for the errors to describe the problem, not the fix - maybe 'this statement is empty' or just 'empty statement'?

The fix message (shown by IDEs) can be specified separately

Reply:
Now: "This statement is empty" and fix message is "Remove statement".